tests/test_pip_chill.py

```diff
@@ -9,151 +9,159 @@
 """
 
 import os
-import sys
-import unittest
 
 from pip_chill import pip_chill
 from pip_chill.pip_chill import Distribution
 
 
-class TestPip_chill(unittest.TestCase):
```

Comment:
This makes pytest a development requirement. I'm not sure I like that. It also requires familiarity with the pytest framework to read the tests whereas the original unittest-based ones don't.